lib/src/protobuf/coded_buffer_writer.dart

```diff
@@ -317,7 +317,8 @@ class CodedBufferWriter {
         _writeVarint32(value ? 1 : 0);
         break;
       case PbFieldType._BYTES_BIT:
-        _writeBytesNoTag(value);
+        _writeBytesNoTag(
+            value is TypedData ? value : Uint8List.fromList(value));
```

Comment:
maybe it is better to still use `new ...` for compatibility purposes.

Reply:
Done